vscode-extension/media/CreateAIConfig.md

```diff
@@ -0,0 +1,5 @@
+## Create New AIConfig
+
+Open the Command Palette (CMD+Shift+P) and run the command `AIConfig: Create New`
```

Comment:
nit, should we say "... run the command `AIConfig: Create New (JSON)` or `AIConfig: Create New (YAML)`"